orcestra_book/intro.md

```diff
@@ -2,16 +2,36 @@
 
 **ORCESTRA - Organized Convection and EarthCare Studies over the Tropical Atlantic**
 
-The overarching objective of ORCESTRA is to better understand the physical mechanisms that organize tropical convection at the mesoscale, influencing the structure and dynamics of the inter-tropical convergence zone (ITCZ). This includes the interaction of convective organization with tropical waves and air-sea interaction, and the impact of convective organization on climate and the Earth’s radiation budget and processes of tropical cyclogenesis. In addition to advancing understanding of tropical meteorology and atmospheric processes, ORCESTRA observations will help calibrate and validate satellite remote sensing (especially EarthCARE) and a new generation of global ocean-eddy and storm-resolving climate models.
 
-ORCESTRA is an international initiative that combines different sub-campaigns:
+The overall objectives of the ORCESTRA campaign are summarised as followed:
+
+```{note}
```

Comment:
I'd recommend to put "overall objectives" as the title of the note. Otherwise, the appearance of the word "Note" in the rendered page is a bit distracting.

````suggestion
```{admonition} overall objectives of the ORCESTRA campaign    
:class: note 
````

Reply:
Sure, sounds reasonable! Thanks <3